.github/workflows/update-3rdparty.yml

```diff
@@ -0,0 +1,254 @@
+name: Update 3rdparty sources
+on:
+  push:
+#     branches:
+#       - 'release-**'
+
+  schedule:
+    - cron:  '0 0 * * fri'
+
+jobs:
+  update-iremapper-source:
```

Comment:
I've split it into separate jobs so they all run asynchronously: https://github.com/Mudlet/Mudlet/actions/runs/57384020